templates/motd.erb

```diff
@@ -10,14 +10,14 @@ LOG OFF IMMEDIATELY if you do not agree to the conditions stated in this
 warning.
 *******************************************************************************
 
-<%= @operatingsystem %> <%= @operatingsystemrelease %> <%= @architecture %>
+<%= @facts['os']['name'] %> <%= @facts['os']['release']['full'] %> <%= @facts['os']['architecture'] %>
 
-FQDN:      <%= @fqdn %> (<%= @ipaddress %>)
-<% if has_variable?("processor0") then -%>
-Processor: <%= @processorcount %>x <%= @processor0 %>
+FQDN:      <%= @facts['networking']['fqdn'] %> (<%= @facts['networking']['ipaddress'] %>)
+<% if @facts['processor0'] then -%>
+Processor: <%= @facts['processorcount'] %>x <%= @facts['processor0'] %>
 <% end -%>
```

Comment:
I guess this should be
```suggestion
Processor: <%= @facts['processors']['count'] %>x <%= @facts['processors']['models'][0] %>
```
(no idea under which circumstances this could be undefined)